clio-integration-test/src/test/resources/docker-compose/docker-compose-jenkins.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  clio-jenkins-integration-test:
+    image: broadinstitute/clio-integration-test:${CLIO_DOCKER_TAG}
+    environment:
+      ELASTICSEARCH_HTTP_HOST: elasticsearch1.gotc-dev.broadinstitute.org
+    network_mode: "host"
+  clio-server:
+    image: broadinstitute/clio-server:${CLIO_DOCKER_TAG}
```

Comment:
The configs are different so I don't think that will work. E.g. this integration-test service has a different ELASTICSEARCH_HTTP_HOST and requires `network_mode: "host"`.